flame_fire_atlas/lib/flame_fire_atlas.dart

```diff
@@ -107,7 +111,9 @@ class FireAtlas {
     final atlas = FireAtlas()
       ..id = json['id']
       ..tileSize = json['tileSize']
-      ..imageData = json['imageData'];
+      ..imageData = json['imageData']
+      ..tileHeight = json['tileHeight']?.toDouble()
+      ..tileWidth = json['tileWidth']?.toDouble();
```

Comment:
Here we can check for the `tileSize` in case the `tileHeight` or  `tileWidget` are nulls, like:

```
      ..tileHeight = json['tileHeight']?.toDouble() ?? json['tileSize']
      ..tileWidth = json['tileWidth']?.toDouble() ?? json['tileSize'];
```

That way we can safely remove the tileSize, and old files will still be read.